### ISSUE TITLE
Condition is not closed on printing

### ISSUE THREAD

**Issue by sabbelasichon** [martin-helmich/typo3-typoscript-parser#52]
When trying to write the typoscript back to the file the end of the conditions is removed when multiple conditions exists

```
[globalVar = GP:foo=1]
    foo = bar
[global]

[globalVar = GP:foo=2]
    foo = bar
[global]
```

results in 

```
[globalVar = GP:foo=1]
    foo = bar

[globalVar = GP:foo=2]
    foo = bar
[global]
```

**Comment by martin-helmich** [comment]
I have to admit that the intricacies of Typoscript escape me at the moment... 🤯  Is this just a code style issue (i.e. are the two snippets functionally identical), or is there a semantical difference between them? IIRC, "nested conditions" are not a thing in TypoScript, aren't they?

Currently, this is kind of by-design, meaning that multiple sequential conditions are considered as multiple `elseif` blocks, instead of separate `if` blocks.

https://github.com/martin-helmich/typo3-typoscript-parser/blob/bc9642076c5a698819f1358047f6c3eb77d9f99a/src/Parser/Printer/PrettyPrinter.php#L72-L81 

TBH, it's probably unavoidable that certain formatting details are lost on parsing -- same goes with indentation, or empty lines (as mentioned in #56), so the printer will most probably always need to impose its own set of code style. But maybe the PrettyPrinter could accept some configuration to make this adjustable?

**Comment by martin-helmich** [comment]
After a quick discussion closed in favor of #59.
